### PR TITLE
Update linux_homework.txt 

### PR DESCRIPTION
**Pull request by SCigiel**
I did my homework with LINUX. This is the first version. 
Autors: 
SCigiel

### CODE REVIEW

**Comment by ziobron** [comment]
@SCigiel, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If any label was added to this PR (like Standard, Premium, Podstawy C++, STL) then everything is fine :)
4. If your solution passed, but nothing happens, modify your code (eg. add an empty line) to run the workflow again.

If you don't know what to do - ask on Discord and provide a link to this PR.

File: homework/linux.txt [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   16  cd backup/
+   17  ls
+   18  cd ..
+   19  ls
+   20  wc -l processes.txt 
+   21  wc -l processes.txt > selected.txt
+   22  ls
+   23  less selected.txt 
+   24  grep -n /bin *
+   25  grep -n /bin * >> selected.txt
```
Comment (ziobron) [inline homework/linux.txt:137]: You should grep in processes.txt file, not in every file in current directory

Comment (SCigiel) [reply homework/linux.txt]: Thanks for advice.

**Comment by ziobron** [comment]
✅ CI Passed - Task: linux
❌ 10 XP not granted.

@SCigiel, this is not important for now. Your solution is correct. Now we will try to automatically map your accounts. If you can see that points are granted in the next message then everything went fine. Otherwise,
please ask @ziobron (Lukin#2054 on Discord) to take a look here. He needs to connect your accounts manually.

**Comment by ziobron** [comment]
✅ CI Passed - Task: linux
🏅 Granted 10 XP for @SCigiel

**Pull request closed without merging**